tsdb/engine/tsm1/file_store_key_iterator.go

```diff
@@ -0,0 +1,111 @@
+package tsm1
+
+import (
+	"bytes"
+	"container/heap"
+)
+
+type keyIterator struct {
+	f    TSMFile
+	p, n int
```

Comment:
Can you add a comment what `p` and `n` are?

Reply:
Assuming `p` is the current index and `n` is the total number of keys?

Reply:
will do 👍🏻

Reply:
done